ajax/libs/crypto-js/package.json

```diff
@@ -1,29 +1,49 @@
 {
   "name": "crypto-js",
-  "filename": "rollups/md5.js",
-  "description": "CryptoJS is a growing collection of standard and secure cryptographic algorithms implemented in JavaScript using best practices and patterns. They are fast, and they have a consistent and simple interface.",
-  "version": "3.1.2",
-  "homepage": "http://code.google.com/p/crypto-js",
+  "title": "crypto-js",
+  "description": "JavaScript library of crypto standards.",
+  "version": "3.1.9",
+  "author": {
+    "name": "Evan Vosberg",
+    "url": "http://github.com/evanvosberg"
+  },
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/brix/crypto-js.git"
+  },
+  "license": "MIT",
   "keywords": [
-    "JavaScript",
-    "Crypto",
+    "security",
+    "crypto",
+    "Hash",
     "MD5",
+    "SHA1",
     "SHA-1",
-    "SHA-2",
-    "SHA-3",
-    "HMAC",
-    "PBKDF2",
-    "AES",
-    "TripleDES",
+    "SHA256",
+    "SHA-256",
+    "RC4",
     "Rabbit",
-    "RC4"
+    "AES",
+    "DES",
+    "PBKDF2",
+    "HMAC",
+    "OFB",
+    "CFB",
+    "CTR",
+    "CBC",
+    "Base64"
   ],
-  "author": {
-    "name": "Jeff Mott"
-  },
-  "license": "BSD-3-Clause",
-  "repository": {
-    "type": "svn",
-    "url": "http://crypto-js.googlecode.com/svn"
+  "filename": "crypto-js.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brix/crypto-js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.+(js|json|map)"
```

Comment:
@evanvosberg CDNJS does not support `json` files yet, let's remove that. Thanks.